pkg/wrangler/context.go

```diff
@@ -121,6 +129,11 @@ type Context struct {
 	TunnelAuthorizer    *tunnelserver.Authorizers
 	PeerManager         peermanager.PeerManager
 	Provisioning        provisioningv1.Interface
+	RBAC                rbacv1.Interface
```

Comment:
This explains why "duplicating" factories is not much overhead https://github.com/rancher/rancher/pull/39876#issuecomment-1358453262. I would rather do it this way than to break the pattern of the interfaces only having functions for each resource controller. This also makes it so that when a user calls this they will only need to call it once on the context when registering their handler.